site/content/docs/5.3/customize/sass.md

```diff
@@ -70,14 +70,15 @@ In your `custom.scss`, you'll import Bootstrap's source Sass files. You have two
 @import "../node_modules/bootstrap/scss/mixins";
 @import "../node_modules/bootstrap/scss/root";
 
-// 6. Optionally include any other parts as needed
+// 6. Optionally include any other parts as needed. More includes can be found in the same directory as these examples.
```

Comment:
```suggestion
// 6. Include any other optional stylesheet partials as desired; list below is not inclusive of all available stylesheets
```

Reply:
Shucks, I don't have edit access so can't merge this suggestion myself.

Reply:
@mdo I've squashed in your suggestion and re-pushed!

Reply:
@mdo Is there anything else you need me to do here?

Reply:
@Kelketek Could you please check the "Allow edits [...] by maintainers" in your PR config as mentioned in https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork. It would help us move forward this PR, rebase whenever needed, etc.

Reply:
Hi @julien-deramond . I don't have the option in my sidebar to allow this, probably because the repo I'm making this PR from is an organizational one and not an individual one. Would you prefer I attempt re-opening this PR from a new individual fork? I have rebased it to the latest master in the meantime.

Reply:
> probably because the repo I'm making this PR from is an organizational one and not an individual one

Yeah, it's probably the reason.

> Would you prefer I attempt re-opening this PR from a new individual fork? I have rebased it to the latest main in the meantime.

I think we're good as you've rebased the branch and included Mark's suggestion. Thanks for that!